docker-compose.yml

```diff
@@ -1,11 +1,10 @@
 version: '3.7'
 services:
+
   bk-register:
     build: bk-register
     restart: always
     init: true
-    networks:
-      - beehive
```

Comment:
i would have just renamed this to `beekeeper` and then you can confidently use the `beekeeper` network name in your `--network` calls to `docker run` without having to rely on how `docker compose` creates the default network names.

Reply:
For the moment I do not see that we need to reference the network at all, but if we need to we can define the name. I think there is only one global definition required in the docker-compose file, it is not necessary do define it for each container individually.